lib/beaker-answers/pe_conf.rb

```diff
@@ -82,6 +82,28 @@ def _generate_1_0_data
         pe_conf["#{ns}::puppetdb_host"] = puppetdb.hostname
       end
 
+      if any_host_with_role?('pe_postgres')
+        pe_conf["#{ns}::database_host"] = the_host_with_role('pe_postgres').hostname
+        if hosts[0].options[:pe_postgresql_options]
+          if hosts[0].options[:pe_postgresql_options][:security] == 'cert'
+            pe_conf["#{ns}::database_ssl"] = true
```

Comment:
this logic is duplicated, consider extracting it out into it's own methods, such as `postgres_cert_answers` and `postgres_password_answers` that returns a hash you can merge into the pe_conf one.